DESIGN.md

```diff
@@ -12,9 +12,9 @@ With some exceptions, the `tune` command calculates settings for each service ba
 
 The ratio, minimum, and maximum vary based upon the PE Infrastructure type and the PE services sharing resources on each PE Infrastructure host.
 
-The supported minimum system resources for the `tune` command are 4 CPU / 8 GB RAM.
+The supported minimum system resources of the Master for the `tune` command is 4 CPU / 8 GB RAM.
 
-#### Standard Reference Architecture (Monolithic Master)
+#### Standard Reference Architecture (Master only)
```

Comment:
This header is documenting, in parens, the role ... not describing the architecture.